pages/place/content/errors.js

```diff
@@ -1,9 +1,9 @@
 module.exports = {
   site: {
-    required: 'Licensed premises all need a site name.'
+    required: 'Select a site for this new approved area.'
   },
   name: {
-    required: 'This could be a room number, like G1 for example.'
+    required: 'Enter a name for this new approved area.'
```

Comment:
What if it's not new?

Reply:
You don't expect me to actually read error messages? Copy paste FTW